clients/feeder/feeder.go

```diff
@@ -128,10 +128,8 @@ func newTestServer(t *testing.T, network *utils.Network) *httptest.Server {
 		assert.Equal(t, []string{"API_KEY"}, r.Header["X-Throttling-Bypass"])
 		assert.Equal(t, []string{"Juno/v0.0.1-test Starknet Implementation"}, r.Header["User-Agent"])
 
-		wd, err := os.Getwd()
 		require.NoError(t, err)
 
-		base := wd[:strings.LastIndex(wd, "juno")+4]
```

Comment:
Also I did a small refactor on the `newTestServer` function. Essentially we shouldn't retrieve the directory path with the hardcoded "juno" string. There are 2 issues with this:

1. The directory name might not even be "juno". Users may choose to set some arbitrary directory name.
2. We have a directory of `juno/cmd/juno`, since `strings.LastIndex` returns the last index at which a substring occurs, it will return `juno/cmd/juno` instead of just `juno`. The simple solution to this is just use `strings.Index`

Both of the above scenarios will cause the tests to fail. The refactor solution is to traverse up the directory tree from the current working directory until the target directory is found, solving the issues above.